universitas-negeri-semarang-fakultas-matematika-dan-ilmu-pengetahuan-alam.csl

```diff
@@ -270,7 +270,6 @@
       <if variable="URL">
         <choose>
           <if type="article-journal article-magazine article-newspaper bill chapter entry-dictionary entry-encyclopedia legislation paper-conference" match="any">
-            <text prefix="[" suffix="]"/>
           </if>
         </choose>
       </if>
```

Comment:
I've looked for it. But, it doesn't exist. Have you deleted it?
Hope this new update works. :)